starter_ai_agents/microsoft_agents_starter/.env.example

```
@@ -0,0 +1 @@
+NEBIUS_API_KEY=your_api_key_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)